micro-rdk-ffi/Cargo.toml

```diff
@@ -40,10 +40,3 @@ cbindgen.workspace = true
 embuild.workspace = true
 serde.workspace = true
 serde_json.workspace = true
-
```

Comment:
@npmenard - I removed this, because I don't think it is required: this crate should obtain these settings automatically from the root crate, which is `micro-rdk-server`.